providers/base/units/miscellanea/jobs.pxu

```diff
@@ -489,10 +489,19 @@ category_id: com.canonical.plainbox::miscellanea
 estimated_duration: 60
 id: miscellanea/ubuntu-desktop-recommends
 requires: package.name == 'ubuntu-desktop'
-command: check-ubuntu-desktop-recommends.sh
+command: check-ubuntu-desktop-recommends.sh ubuntu-desktop
 _summary: Check if Debian packages in Recommends of ubuntu-desktop are installed
 _description: Check if Debian packages in Recommends of ubuntu-desktop are installed
 
+plugin: shell
+category_id: com.canonical.plainbox::miscellanea
+estimated_duration: 60
+id: miscellanea/ubuntu-desktop-minimal-recommends
+requires: package.name == 'ubuntu-desktop-minimal'
+command: check-ubuntu-desktop-recommends.sh ubuntu-desktop-minimal
+_summary: Check if Debian packages in Recommends of ubuntu-desktop-minimal are installed
+_description: Check if Debian packages in Recommends of ubuntu-desktop-minimal are installed
```

Comment:
Since it's a copy/paste of the summary, it's not really useful. Please consider removing it (and the same should apply to `miscellanea/ubuntu-desktop-recommends` as well)